app/views/submissions/index.html.haml

```diff
@@ -36,7 +36,11 @@
             - else
               = submission.editor
           %td= submission.received.strftime("%d-%m-%Y")
-          %td= submission.last_deadline
+          %td    
+            - if submission.last_review != nil && submission.last_review.missed_deadline
```

Comment:
Proszę też popatrzeć jak to zmieniłem w [swojej poprawce](https://github.com/apohllo/rocznik/commit/3ef9b7520f2864e08d26d7e87a0409090d45886c#diff-512fa9b87ca920e819cc32db6c74cfb9R96)